README.md

```diff
@@ -64,7 +62,8 @@ extension IPOrigin: JSONMappable {
     }
 }
 
-let request = NetworkRequest(path: "/ip", baseURLKey: baseURLKey)
+let url = URL(string: "https://httpbin.org/ip")!
```

Comment:
When looking into the documentation, I can't find a definition of an inizializer on URLRequest, that takes a baseURL and a path. Does it really exist?